### PR TITLE
Enhancement: Rework /tools directory

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

File: tools/composer.json [hunk 1]
```diff
@@ -3,6 +3,6 @@
         "gansel-rechtsanwaelte/php-cs-fixer-config": "^1.1.0"
     },
     "require": {
-        "php": "^7.2"
+        "php": ">=7.4"
```
Comment (localheinz) [inline tools/composer.json:6]:
```suggestion
        "php": "^7.4 || ^8.0"
```

**Review by localheinz — APPROVED** [review]
👍

**Review by localheinz — CHANGES_REQUESTED** [review]
A job is failing!

🙈

**Comment by OskarStark** [comment]
Fixed by using a newer php-cs-fixer, but this is also installed via our package, right? So we should use it from the tools/vendor dir

**Review by localheinz — APPROVED** [review]
👍

**Pull request merged**